src/QuickMarcEditor/utils.test.js

```diff
@@ -1051,6 +1052,15 @@ describe('QuickMarcEditor utils', () => {
         tag: '005',
         id: uuid(),
       }, {
+        tag: '008',
+        id: uuid(),
+        content: HOLDINGS_FIXED_FIELD_DEFAULT_VALUES,
+      }, {
+        tag: '852',
+        id: uuid(),
+        content: '$a',
+      },
+      {
```

Comment:
Minor thing, but maybe it's better to keep the same format as above.

```suggestion
      }, {
```